src/Atmos/Model/refstate.jl

```diff
@@ -0,0 +1,74 @@
+### Reference state
+
+abstract type AbstractReferenceState{T} end
+struct ReferenceState{T} <: AbstractReferenceState{T}
+  T_ground::T
+  P_ground::T
+  q_tot_ground::T
+end
+
+vars_state(m::AbstractReferenceState    , DT) = @vars()
+vars_gradient(m::AbstractReferenceState , DT) = @vars()
+vars_diffusive(m::AbstractReferenceState, DT) = @vars()
+vars_aux(m::AbstractReferenceState      , DT) = @vars(ρ::DT, p::DT, T::DT)
```

Comment:
Should `p` be in the reference state of `ρe`? Seems that it would make more sense to match that name here to whats in the [`AtmosModel.jl`](https://github.com/climate-machine/CLIMA/blob/52d173ce5eeb0917906990128529fd144f82ec65/src/Atmos/Model/AtmosModel.jl#L38-L46)

Reply:
My thinking here is when the full state is needed the reference state can just be added.

Reply:
Good point. I'm not sure how to answer that. In the EDMF model, we've typically only used the reference `ρ, p, T` (maybe this is due to available measurements?). We can certainly add these fields if it makes sense to mirror `AtmosModel`.

Reply:
Adding `ρe` makes sense--we'll need it.

Reply:
You'd also want to add a `q_tot` that corresponds to a fixed and prescribed relative humidity RH, as described in the docs. We'll needs this for spinning up moist models (otherwise it takes years of simulated time to moisten the stratosphere).

Reply:
Ok, I'll make sure to add `ρe` when updating according to the new docs. And I'll take a look at the RTB case.